### PR TITLE
Change V1ALPHA2_SWITCH variable to CAPI_VERSION

### PR DESCRIPTION
**Pull request by maelk**
in order to support v1alpha3

### CODE REVIEW

File: lib/common.sh [hunk 1]
```diff
@@ -99,8 +99,13 @@ export BAREMETAL_OPERATOR_IMAGE=${BAREMETAL_OPERATOR_IMAGE:-"quay.io/metal3-io/b
...
 export OPENSTACK_CONFIG=$HOME/.config/openstack/clouds.yaml
 
-# v1alpha2 var
+# CAPI version
 export V1ALPHA2_SWITCH=${V1ALPHA2_SWITCH:-"false"}
+if [ "${V1ALPHA2_SWITCH}" == true ]; then
+  export CAPI_VERSION=${CAPI_VERSION:-"v1alpha2"}
+else
+  export CAPI_VERSION=${CAPI_VERSION:-"v1alpha2"}
+fi
```
Comment (fmuyassarov) [inline lib/common.sh]: Isn't it setting the same value for CAPI_VERSION no matter if condition met or not?

Comment (maelk) [reply lib/common.sh]: good catch, fixed!

**Pull request merged**